### PR TITLE
Fix : Keep underMaintenance active between search, reloads, filter change

### PR DESCRIPTION
**Pull request by cballevre**
```
### 🐛 Bug Fixes

* Show active categories with other filters than categories
* Keep underMaintenance active between search, reloads, filter change
```

### CODE REVIEW

File: src/ducks/apps/components/Sections/Sections.jsx [hunk 1]
```diff
@@ -27,53 +27,51 @@ import Filters from './components/Filters'
...
-
-  const onSearchChanged = useMemo(
-    () =>
-      debounce(value => {
-        const fuse = new Fuse(appsForSearch, options)
-        setSearchResults(value ? fuse.search(value) : null)
-      }, 300),
-    [appsForSearch, options]
-  )
+    const fuse = new Fuse(filteredApps, options)
```
Comment (Crash--) [inline src/ducks/apps/components/Sections/Sections.jsx:59]: I know that this is not part of this PR, but this is weird, no? We should instanciate only once the `Fuse` object with all the apps, and then make a search on the filter and on the searchValue. We should not create this `Fuse` object every time, it doesn't make sense and should consume a lot of RAM.

**Review by Crash-- — APPROVED** [review]
I don't really like this solution, but let's move on.

File: src/ducks/apps/components/Sections/Sections.jsx (hunk 1 above)
Comment (cballevre) [reply src/ducks/apps/components/Sections/Sections.jsx]: Yes it's weird now you have point out. There is an other thing that disturb me, the filters are applied twice once before the search and once app list. In the same refacto, maybe we can use fuse.js for the two.
